frontend/src/container/FormAlertChannels/index.tsx

```diff
@@ -42,9 +42,13 @@ function FormAlertChannels({
 }: FormAlertChannelsProps): JSX.Element {
 	const { t } = useTranslation('channels');
 	const isUserOnEEPlan = useFeatureFlags(FeatureKeys.ENTERPRISE_PLAN);
-	const hasFeature = useFeatureFlags("ALERT_CHANNEL_".concat(type.toUpperCase()));
+	const hasFeature = useFeatureFlags(
+		'ALERT_CHANNEL_'.concat(
```

Comment:
can we have typescript util which check the feature flag rather than enforcing a type would recommend to have util for this one

Reply:
not sure what you mean by"rather than enforcing a type". Please share a sample steps so i am clear on the ask.

Reply:
sorry @mindhash `isTraceFilterEnum` is one good example that you can refer here